integration-tests/cucumber-tests-platform-distributionautomation/src/test/java/org/opensmartgridplatform/cucumber/platform/distributionautomation/glue/steps/database/FeederSteps.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2021 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.cucumber.platform.distributionautomation.glue.steps.database;
+
+import java.util.Map;
+
+import org.opensmartgridplatform.adapter.kafka.da.domain.entities.Feeder;
+import org.opensmartgridplatform.adapter.kafka.da.domain.entities.Location;
+import org.opensmartgridplatform.adapter.kafka.da.domain.repositories.FeederRepository;
+import org.opensmartgridplatform.cucumber.core.ReadSettingsHelper;
+import org.opensmartgridplatform.cucumber.platform.distributionautomation.PlatformDistributionAutomationDefaults;
+import org.opensmartgridplatform.cucumber.platform.distributionautomation.PlatformDistributionAutomationKeys;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import io.cucumber.java.en.Given;
+
+public class FeederSteps {
+
+    @Autowired
+    private LocationSteps locationSteps;
+
+    @Autowired
+    private FeederRepository feederRepository;
+
+    @Given("a feeder")
+    public void givenAFeeder(final Map<String, String> settings) {
+        final Feeder feeder = new Feeder();
+        final Location location = this.locationSteps.findLocation(
+                ReadSettingsHelper.getString(settings, PlatformDistributionAutomationKeys.SUBSTATION_IDENTIFICATION,
+                        PlatformDistributionAutomationDefaults.SUBSTATION_IDENTIFICATION));
```

Comment:
I'm also not so happy about the naming, especially introducing substations in gxf code. 
(In my opinion it would probably be more appropriate to use device instead of substation here).
For now I've kept it as it was...